setup.py

```diff
@@ -72,7 +72,7 @@ def get_extras_require() -> Dict[str, List[str]]:
             "pytorch-ignite",
             "scikit-image",
             "scikit-learn",
-            "torch>=1.4.0+cpu",
+            "torch==1.4.0+cpu",
             "torchvision>=0.5.0+cpu",
```

Comment:
It maybe out of score in this PR but it could be better to use `==` for `torchvision` as well.
(Although I'm not sure whether it matters, `torchvision>=0.5.0+cpu` install torchvision with CUDA)

- `torchvision>=0.5.0+cpu`

```
Collecting torchvision>=0.5.0+cpu
507
  Downloading https://download.pytorch.org/whl/cu92/torchvision-0.5.0%2Bcu92-cp37-cp37m-linux_x86_64.whl (3.9 MB)
```

- `torchvision==0.5.0+cpu'

```
$ pip install 'torchvision==0.5.0+cpu' -f https://download.pytorch.org/whl/torch_stable.html
Looking in links: https://download.pytorch.org/whl/torch_stable.html
Collecting torchvision==0.5.0+cpu
```

https://github.com/hvy/optuna/pull/5/checks?sha=306b2e89defadc4685ae860ae64c05e99b4a0ead

Reply:
Updated `torchvision` as well. Thanks for your suggestion.

Reply:
```
Collecting torchvision==0.5.0+cpu
  Downloading https://download.pytorch.org/whl/cpu/torchvision-0.5.0%2Bcpu-cp37-cp37m-linux_x86_64.whl (5.4 MB)
```

https://app.circleci.com/pipelines/github/optuna/optuna/40/workflows/1a206914-5195-4fc4-ae67-e0434c7f4999/jobs/36754

:+1: